PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+- Please include a summary of the change and which issue is fixed. 
+- Please also include relevant motivation and context. 
+- List any dependencies that are required for this change.
+
+## User Story / Task Link
+
+Addresses [AB#XXXXX](url)
```

Comment:
url is automatically updated when azure-bot encounters AB# syntax. The extra markdown syntax might lead people to think they have to go find the link themselves.